.mergify.yml

```
@@ -170,8 +170,8 @@ pull_request_rules:
           This pull request does not have a backport label. Could you fix it @{{author}}? 🙏
           To fixup this pull request, you need to add the backport labels for the needed
           branches, such as:
-          * `backport-7.x` is the label to automatically backport to the `7.x` branch.
-          * `backport-7./d` is the label to automatically backport to the `7./d` branch. `/d` is the digit
+          * `backport-7.17` is the label to automatically backport to the 7.17 LTS branch.
```

**question**: Should we refer to 7.17 as LTS ? I can see the reasoning but it's the first time I've seen it used in the repo for that branch

---

I wrongly assumed 7.17 = LTS, let me remove it 

```suggestion
          * `backport-7.17` is the label to automatically backport to the 7.17 branch.
```